ui/apos/components/AposDuplicateImportModal.vue

```diff
@@ -0,0 +1,331 @@
+            <AposButton
+              class="apos-context-menu__btn"
+              data-apos-test="contextMenuTrigger"
+              :label="checked.length ? 'apostrophe:deselectAll' : 'apostrophe:selectAll'"
+              type="quiet"
+              role="button"
+              @click.stop="deselect"
+            />
+
+            <div class="apos-import-duplicate__separator" />
+            <div class="apos-import-duplicate__docs-list">
+              <AposCheckbox
+                v-for="doc in duplicatedDocs"
+                :key="doc.aposDocId"
+                v-model="checked"
+                tabindex="-1"
+                :choice="{
+                  value: doc.aposDocId,
+                  label: doc.title
+                }"
+                :field="{
+                  label: doc.title,
+                  name: doc.aposDocId
+                }"
+              />
+            </div>
+          </div>
+
+          <div class="apos-import-duplicate__separator apos-import-duplicate__separator--full-width" />
+
+          <div class="apos-import-duplicate__btns">
+            <AposButton
+              class="apos-import-duplicate__btn"
+              label="apostrophe:cancel"
+              @click="cancel"
+            />
+            <AposButton
+              ref="runOverride"
+              class="apos-import-duplicate__btn"
+              :label="$t('aposImportExport:importDuplicateContinue')"
+              type="primary"
+              @click="runOverride"
+            />
+          </div>
+        </template>
+      </AposModalBody>
+    </template>
+  </AposModal>
+</template>
+
+<script>
+export default {
+  props: {
+    type: {
+      type: String,
+      required: true
+    },
+    duplicatedDocs: {
+      type: Array,
+      required: true
+    },
+    exportPath: {
+      type: String,
+      required: true
+    },
+    importedAttachments: {
+      type: Array,
+      required: true
+    },
+    jobId: {
+      type: String,
+      required: true
+    },
+    notificationId: {
+      type: String,
+      required: true
+    }
+  },
+
+  emits: [ 'safe-close', 'change' ],
+
+  data() {
+    return {
+      modal: {
+        active: false,
+        type: 'overlay',
+        showModal: false,
+        disableHeader: true
+      },
+      checked: [],
+      exportFileCleaned: false
+    };
+  },
+  computed: {
+    moduleLabel() {
+      const moduleOptions = apos.modules[this.type];
+      const label = moduleOptions.pluralLabel;
+
+      return this.$t(label).toLowerCase();
+    }
+  },
+
+  async mounted() {
+    this.modal.active = true;
+    this.checked = this.duplicatedDocs.map(({ aposDocId }) => aposDocId);
+  },
+
+  methods: {
+    async closeModal() {
+      if (!this.exportFileCleaned) {
+        await this.cleanExportFile();
+      }
+      this.$emit('safe-close');
+    },
+    async cleanExportFile() {
+      try {
+        await apos.http.post('/api/v1/@apostrophecms/import-export/clean-export', {
```

Comment:
you should have a look to `getBrowserData` `action`

Reply:
there are no action for modules that aren't from piece-type.